<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPowerCheckDialog</class>
 <widget class="QDialog" name="CPowerCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_power_normal">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>功率正常</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_3" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_power_abnormal">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_top" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_1" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pre_page">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next_page">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return_back">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_7" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_19">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_14">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_20">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_2" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_21">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_16">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_4" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>20</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_22">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_17">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_5" native="true">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_23">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_11">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_18">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_6" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_24">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_12">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_m_a">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_m_a">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>30</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_11">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_12">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_13">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_14">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_15">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_16">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_17">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_18">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_19">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_20">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_21">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_22">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_23">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_24">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_power_normal">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_power_innormal">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_m_b">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>30</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_m_b">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_m_c">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>30</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_m_c">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_25">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_25">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_29">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_26">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_27">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_28">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_29">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_27">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_w_30">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>支路24</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_30">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_26">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_l_w_28">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMyQPushButton</class>
   <extends>QPushButton</extends>
   <header>keyboard/myqpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
